### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
List of changes:
- Fix typo
- Add changelog to html doc
- Add Changelog link to hex info page

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Stanislav Vishnevskiy
+Copyright (c) 2014-present, Stanislav Vishnevskiy
```
Comment (mitchellhenke) [inline LICENSE:3]: I'd have to double check on this, could you remove the change for now?

**Review by mitchellhenke — CHANGES_REQUESTED** [review]
One small change, otherwise looks great

Thank you so much!

File: LICENSE (hunk 1 above)
Comment (kianmeng) [reply LICENSE]: > I'd have to double check on this, could you remove the change for now?

Done.

Comment (kianmeng) [reply LICENSE]: @mitchellhenke It seems that the use of copyright year `2014-present` is not recommended, see https://github.com/riverrun/comeonin/pull/153#issuecomment-757405132.

Do let me know if you found something on this as well. Thanks. :smile:

Comment (mitchellhenke) [reply LICENSE]: Thanks for the reference, everything looks great now 🙂

**Review by mitchellhenke — APPROVED** [review]
Thank you so much for submitting these improvements!

**Pull request merged**